packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -252,17 +252,16 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
     <div ref={containerRef} className={mergeStyles(containerDivStyles)}>
       <Stack verticalFill grow styles={compositeOuterContainerStyles} id={compositeParentDivId}>
         <Stack horizontal grow>
-          {!isMobileWithActivePane && (
```

Comment:
Can you test the incoming and outgoing video tiles in the PiPiP works as expected on mobile? i.e. start a call with another participant, turn on video for both, on one that is mobile open the pane and check the videos show correctly, then close the pane and check video gallery tiles are correct (Want to test in case the call video gallery might still have a hold to the video streams if it is not unmounted)

Reply:
(we just unified all the logic surrounding creating and disposing streams so hopefully this won't be an issue anymore)

Reply:
Good Point. Let me test that

Reply:
Confirmed. Working as expected. :)